src/loaders/genome_collection/compute_genome_attribs.py

```diff
@@ -70,6 +71,8 @@
 from rpy2 import robjects
 
 from src.loaders.common import loader_common_names
+from src.service.data_products.heatmap_common_models import (Cell, ColumnInformation, ColumnCategory, HeatMapMeta,
```

Comment:
If these are going to be shared (which I think it right) they should be moved form service to common

Reply:
Although maybe they should be used in the parser rather than here

Reply:
👍 will move the parsing logic to the parsing script in next PR.

Reply:
What exactly do you mean? I think the vast majority of the logic should stay here, but maybe it just outputs csv or json without going through the models?

Reply:
Ok, I see you already made a commit, if that stuff moves to the parser script that seems fine, as long as we're done with the R stuff. Easier to deal with the unique cell IDs if you handle it in the parser script as well

Reply:
:+1: